CI_BUILDERS.md

```diff
@@ -16,7 +16,7 @@ Many of the other arguments should be reasonably self-explanatory and its likely
 `CI_BUILD_ARGS` and `CI_TEST_ARGS` are basically passed directly to colcon build and test respectively so you should be familiar with those.
 
 Our CI has limited capacity, and the default `ros2.repos` file is constantly growing.
-To save build time limit the scope CI builds to only packages that are changed by the pull request being tested or are likely to be affected by the changes. 
+To save build time limit the scope CI builds to only packages that are changed by the pull request being tested or are likely to be affected by the changes.
```

Comment:
This change is spurious and I'd be very tempted to split it out into its own PR. Especially when squashing the PR overall since the commit can neither be marked as solely cosmetic nor specific to a single logical change.

Reply:
For this particular repository, I actually think these kinds of changes are fine to do inline with other ones (as long as they are small).  That is, it is easy to determine that this is a cosmetic change, and we are never going to be backporting these changes to any other branches.  A PR just to do these small cleanups seems like overkill to me.

That said, this is not a hill I'm going to die on, so I'll go ahead and remove these for now.